documentation/src/docs/deployment/kubernetes-install.md

```diff
@@ -0,0 +1,417 @@
+<!--
```

Comment:
If I'm not wrong, this is just the content that can be found in https://github.com/pravega/pravega-operator. The problem of copying it here is that we will need to update this page every time that the operator changes, which is not good for maintainability. I suggest to delete this page and just link the `Kubernetes` entry in table of contents to the appropriate README version of the pravega-operator repository.

Reply:
@RaulGracia  No Raul few sections have been removed and inclusions has been made. So for time being we will link the way as how i have framed.... Sooner once the documentation is all set in the Pravega operator will refer separately from `Pravega operator repo`.